.azure-pipelines/ci.yml

```diff
@@ -59,6 +59,7 @@ steps:
     done
     mysql -e "CREATE USER ispyb_api@'%' IDENTIFIED BY 'password_1234'; GRANT ispyb_processing to ispyb_api@'%'; GRANT ispyb_import to ispyb_api@'%'; SET DEFAULT ROLE ispyb_processing FOR ispyb_api@'%';"
     mysql -e "CREATE USER ispyb_api_future@'%' IDENTIFIED BY 'password_4321'; GRANT SELECT ON ispybtest.* to ispyb_api_future@'%';"
+    mysql -e "CREATE USER ispyb_api_sqlalchemy@'%' IDENTIFIED BY 'password_5678'; GRANT SELECT ON ispybtest.* to ispyb_api_sqlalchemy@'%'; GRANT INSERT ON ispybtest.* to ispyb_api_sqlalchemy@'%'; GRANT UPDATE ON ispybtest.* to ispyb_api_sqlalchemy@'%';"
```

Comment:
Should this be read-only?

Reply:
This is required to insert data for testing the ProcessingJob* tables - yes, I could have used the existing stored procedures, however I thought it useful/valuable to test the sqlalchemy interface for inserting data as well as reading. The above permissions are only relevant in the context of the CI job anyway, and have no impact on outside behaviour.

https://github.com/DiamondLightSource/ispyb-api/pull/123/files#diff-3eee5431b0f1a98627dd78c8f328216c7cef8fc5e6f9bc9a003c983583c94ee4R75-R96